docs/setup.md

```diff
@@ -38,6 +39,8 @@ You can also contact us on slack for any further queries.
 - If you face the error that `Sorry! Your operating is not supported by this script`
   - You can try running the system dependent scripts from setup directory, according to your system
   - You can try manual installation
+- After running 'gulp', if you face the error `Failed to load external module @babel/register`
```

Comment:
Instead of putting `--ignore-engines` in the documentation here, the setup script itself should be changed to use that flag.

Also, the manual setup instructions below should be updated to include that flag.